files/en-us/web/css/css_fonts/opentype_fonts_guide/index.html

```diff
@@ -151,9 +151,9 @@ <h3 id="Font_variant_shorthand_Cssxreffont-variant">Font variant shorthand ({{Cs
 
 <h2 id="Font_feature_settings">Font feature settings</h2>
 
-<p>{{cssxref("font-feature-settings")}} is the 'low level syntax' that allows explicit access to every named available OpenType feature. This gives a lot of control, but has some disadvantages in how it impacts inheritance and — as mentioned above — if you wish to change one setting, you have to redeclare the entire string (unless you're using <a href="/en-US/docs/Web/CSS/Using_CSS_variables">CSS custom properties</a> to set the values). Because of this, it's best to use the standard properties shown above wherever possible.</p>
+<p>{{cssxref("font-feature-settings")}} is the 'low level syntax' that allows explicit access to every named available OpenType feature. This gives a lot of control but has some disadvantages in how it impacts inheritance and — as mentioned above — if you wish to change one setting, you have to redeclare the entire string (unless you're using <a href="/en-US/docs/Web/CSS/Using_CSS_variables">CSS custom properties</a> to set the values). Because of this, it's best to use the standard properties shown above wherever possible.</p>
 
-<p>There a huge number of possible features. You can see examples of a number of them above, and there are several resources available for finding more of them.</p>
+<p>There are a huge number of possible features. You can see examples of a number of the above, and there are several resources available for finding more of them.</p>
```

Comment:
them sounds corrrect to me. Why the _the?

Reply:
Agree, though perhaps "examples **for** a number of them"

Reply:
The word <b>them</b> doesn’t seem to fit this context. Consider replacing it with the.

Reply:
Sounds grammatically correct to both Hamish and me…
```suggestion
<p>There are a huge number of possible features. You can see examples of a number of them above, and there are several resources available for finding more of them.</p>
```